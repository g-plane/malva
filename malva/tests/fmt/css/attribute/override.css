/*cfg quotes = "alwaysSingle"
"attrSelector.quotes" = "alwaysDouble"*/
[key=value], [key='value'], [key="value"] {
  content: '';
}
