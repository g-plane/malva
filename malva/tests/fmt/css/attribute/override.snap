---
source: malva/tests/fmt.rs
---
/*cfg quotes = "alwaysSingle"
"attrSelector.quotes" = "alwaysDouble"*/
[key="value"], [key="value"], [key="value"] {
  content: '';
}
